RELEASE_PROCESS.md

```diff
@@ -0,0 +1,186 @@
+# Release Process
+
+- [Release Process](#release-process)
+  - [Major Release Procedure](#major-release-procedure)
+    - [Changelog](#changelog)
+    - [Release Notes](#release-notes)
+    - [Tagging Procedure](#tagging-procedure)
+      - [Test building artifacts](#test-building-artifacts)
+      - [Installing goreleaser](#installing-goreleaser)
+  - [Non-major Release Procedure](#non-major-release-procedure)
+  - [Major Release Maintenance](#major-release-maintenance)
+  - [Stable Release Policy](#stable-release-policy)
+
+
+This document outlines the release process for Cosmos Hub (Gaia).
+
+Gaia follows [semantic versioning](https://semver.org), but with the following deviations to account for state-machine and API breaking changes: 
+
+- State-machine breaking changes will result in an increase of the major version X (X.y.z).
+- API breaking changes will result in an increase of the minor version Y (x.Y.z | x > 0).
```

Comment:
API here is anything that breaks how you use the library code (func signatures, type defs etc)

Reply:
@smarshall-spitzbart Changed the paragraf to:
```

Gaia follows [semantic versioning](https://semver.org), but with the following deviations to account for state-machine and API breaking changes: 

- State-machine breaking changes will result in an increase of the major version X (X.y.z).
- Emergency releases & API breaking changes (changes in node interactions e.g. queries) will result in an increase of the minor version Y (x.Y.z | x > 0).
- All other changes will result in an increase of the patch version Z (x.y.Z | x > 0).

**State compatibility**: 
It is critical for the patch releases to be state-machine compatible with prior releases in the same minor version. 
For example, v9.2.1 must be compatible with v9.2.0.

Minor releases will be handled on a case-by-case basis, but generally should only arise in case of safety or security issues that require a co-ordinated network upgrade without a governance process.
```

Please review as soon as possible, the meat of this PR is the new release tooling. Any documentation can be addressed in a follow-up PR.